doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -231,7 +250,25 @@ The following table indicates the software maturity levels of the support for ea
 
 .. toggle::
 
-  .. sml-table:: wifi
+   +-----------------------------------------------+----------+----------+----------+----------+----------+-------------+------------------+----------------+----------------+---------+----------------+----------------+
+   | Feature                                       | nRF52810 | nRF52811 | nRF52820 | nRF52832 | nRF52833 | nRF52840    | nRF5340          | nRF54H20       | nRF54L15       | nRF9151 | nRF9160        | nRF9161        |
+   +===============================================+==========+==========+==========+==========+==========+=============+==================+================+================+=========+================+================+
+   | **Bluetooth LE Coexistence**                 | -        | -        | -        | -        | -        | Experimental| Supported<sup>1</sup>| -            | -            | -       | -              | -              |
+   +-----------------------------------------------+----------+----------+----------+----------+----------+-------------+------------------+----------------+----------------+---------+----------------+----------------+
+   | **Thread Coexistence**                        | -        | -        | -        | -        | -        | -           | Experimental     | -              | -              | -       | -              | -              |
```

Comment:
This is not visible in the Documentation Link generated for this PR. Will this be seen in the documentation for main build?